src/plugins/controls/public/time_slider/components/anchored_range.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { EuiRange, EuiRangeTick } from '@elastic/eui';
+import { _SingleRangeChangeEvent } from '@elastic/eui/src/components/form/range/types';
+
+interface Props {
+  value: [number, number];
+  onChange: (value?: [number, number]) => void;
+  stepSize: number;
+  ticks: EuiRangeTick[];
+  timeRangeMin: number;
+  timeRangeMax: number;
+}
+
+export function AnchoredRange(props: Props) {
```

Comment:
**Nit:** In our team's style guide (I can send it your way if you don't have it), we opted to prefix all of the components with the specific "domain" they belong to (which, in the controls context, applies specifically to the control type) - for example, following this style, this would be called `TimeSliderAnchoredRange` and the filename would be changed accordingly to `time_slider_anchored_range.tsx`. 

Both the range slider and the options list **should** be following this so, for the sake of consistency, I think it would be good to follow this for the time slider too :+1: